dandi/metadata/util.py

```diff
@@ -840,6 +840,36 @@ class Neurodatum(TypedDict):
         "technique": None,
         "approach": "optogenetic approach",
     },
+    "PoseEstimationSeries": {
+        "module": "ndx-pose",
```

Comment:
```suggestion
        "module": "behavior",
```

@rly As I understand, this is all that is necessary to get this through? (and also applying to the others)